content/academy/data_collection_basics/node_js_scraper.md

```diff
@@ -12,31 +12,31 @@ Finally we have everything ready to start scraping! Yes, the setup is a bit daun
 
 ## [](#downloading-html) Downloading HTML
 
-We will use the `got-scraping` library to download the HTML of the <a href="https://www.alexa.com/topsites" target="_blank">Alexa Top Sites index</a>. Careful, the `import` statement is a little different than in the previous chapter.
+We will use the `got-scraping` library to download the HTML of <a href="https://commerce-qd83plqbj-mstephen19.vercel.app/search/on-sale" target="_blank">Morgan Webstore's</a> on-sale products page. Careful, the `import` statement is a little different than in the previous chapter.
 
-```js
+```JavaScript
 // main.js
 import { gotScraping } from 'got-scraping';
 
-const response = await gotScraping('https://www.alexa.com/topsites');
+const response = await gotScraping('https://commerce-qd83plqbj-mstephen19.vercel.app/search/on-sale');
 const html = response.body;
 console.log(html);
 ```
 
-Now run the script (using `node main.js`). After a brief moment, you should see the page's HTML printed to your terminal. If you get an error that says something along the lines of **urlToHttpOptions is not a function**, you need to update Node.js to version 15.10 or higher. If you followed the installation instructions earlier, you don't need to worry about this, because you have the correct version installed.
+Now run the script (using `node main.js`). After a brief moment, you should see the page's HTML printed to your terminal. If you get an error that says something along the lines of `urlToHttpOptions is not a function`, you need to update Node.js to version 15.10 or higher. If you followed the installation instructions earlier, you don't need to worry about this, because you have the correct version installed.
 
-> `gotScraping` is an `async` function and the `await` keyword is used to pause execution of the script until it returns the `response`. <a href="https://javascript.info/async-await" target="_blank">Learn more</a> about `async` and `await`.
+> `gotScraping` is an `async` function and the `await` keyword is used to pause execution of the script until it returns the `response`. Learn more about `async` and `await` <a href="https://javascript.info/async-await" target="_blank">here</a>.
```

Comment:
I think this is just what we do. Putting the link on Learn more. @m-murasovs made me do it.

Reply:
No need for the HTML link, as the external links get opened in a new tab anyway, so just this will do
```
[Learn more about `async` and `await`](https://javascript.info/async-await)
```